lib/src/chart/line_chart/line_chart_data.dart

```diff
@@ -198,4 +204,27 @@ class FlDotData {
     this.dotSize = 4.0,
     this.checkToShowDot = showAllDots,
   });
-}
\ No newline at end of file
+}
+
+class HorizontalLine extends FlLine {
+  final double x;
+
+  const HorizontalLine({
+    @required this.x,
+    Color color = Colors.black,
+    double strokeWidth = 1,
+  }) : super(color: color, strokeWidth: strokeWidth);
+}
+
+/// This class holds data about drawing chart annotations (data decorations) such as average line and data point lines
```

Comment:
Update the comment, ( we don't need to mention anything about annotation, this word is specific for your use case)

Reply:
Fixed